src/pages/Pools/Home/MembersList/index.tsx

```diff
@@ -86,7 +86,7 @@ export const MembersListInner = ({
 
   // configure list when network is ready to fetch
   useEffect(() => {
-    if (isReady && activeEra.index !== 0 && fetched === 'unsynced') {
+    if (isReady && !activeEra.index.isZero() && fetched === 'unsynced') {
```

Comment:
`!activeEra.index.isZero()` is a bit hard to read and is being used in a few places. I think a new utility, `isNotZero()`, would be easier to read.